deployment/src/main/java/io/quarkiverse/jasperreports/deployment/JasperReportsProcessor.java

```diff
@@ -3,10 +3,7 @@
 import static io.quarkus.deployment.pkg.PackageConfig.JarConfig.JarType.UBER_JAR;
 import static org.jboss.jandex.AnnotationTarget.Kind.CLASS;
 
-import java.io.ByteArrayOutputStream;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
Can these be re-expanded, please?

Reply:
That part was autoformatted by my Ide.

Reply:
I'll update the PR, after I see the builds pass.

Reply:
@bulldog98 if Ubuntu passes you are good.  Its MacOS that takes forever so if Ubuntu passes its good enough not to wait.